public/js/members.js

```diff
@@ -4,4 +4,11 @@ $(document).ready(() => {
   $.get("/api/user_data").then(data => {
     $(".member-name").text(data.email);
   });
+
+  const logout = $("button#logout");
+
```

Comment:
Wow, there isn't more to logging out than this?  I'm VERY skeptical since the logic should involve abandoning the session which would require some sort of interaction with the server by a route.  This is simply a navigation.